.github/workflows/on-prerelease.yml

```diff
@@ -43,10 +43,10 @@ jobs:
           $apiVersion = $(minver -t $apiPrefix)
 
           # Full release name
-          "::set-output name=admin-api::$apiVersion" | Write-Output
+          echo "name=admin-api::$apiVersion" >>$GITHUB_OUTPUT
 
           # SemVer
-          "::set-output name=admin-api-semver::$($apiVersion -Replace $apiPrefix)" | Write-Output
+          echo "name=admin-api-semver::$($apiVersion -Replace $apiPrefix)" >>$GITHUB_OUTPUT
```

Comment:
@jpardogrowthaccelerationpartners 
Update line 49 to "name=admin-api-semver::$($apiVersion -Replace $apiPrefix)" >> $env:GITHUB_OUTPUT (powershell specific)

Reply:
fixed